src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionAboutToSubmitHandler.java

```diff
@@ -79,6 +97,11 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             if (sscsCaseData.getSscsFinalDecisionCaseData().getWriteFinalDecisionPreviewDocument() != null) {
                 createFinalDecisionNoticeFromPreviewDraft(preSubmitCallbackResponse);
```

Comment:
Optional: Use isNotNull() instead of != null?

Reply:
Didn't get which function it is.. Is it from Objects class. Chose to stick with the plain null comparision

Reply:
Sorry, my bad, it's Objects.nonNull()

Optional point, up to you.